### DISCUSSION TITLE
3D files

### DISCUSSION THREAD

**Discussion by stayupthetree** [MLB-LED-Scoreboard/mlb-led-scoreboard#512] [General]
I made this a couple of years ago for a friend and am getting ready to make another. I am not seeing the 3d printer files anymore for the case. Are they still available?

**Comment by ty-porter** [comment]
We do not maintain any resources relating to hardware beyond general instructions for setup.

Other projects may include those, but we have no control over them. You may be referring to the NHL scoreboard's wiki: https://github.com/riffnshred/nhl-led-scoreboard/wiki/3D-printed-enclosures
